sdk/ai/test-resources.json

```diff
@@ -0,0 +1,269 @@
+                                    "privateEndpointNicResourceId": {
+                                        "value": "[concat('/subscriptions/', parameters('privateEndpoints')[copyIndex()].privateEndpointConfiguration.subscription.subscriptionId, '/resourceGroups/', parameters('privateEndpoints')[copyIndex()].privateEndpointConfiguration.resourceGroup.value.name, '/providers/Microsoft.Network/networkInterfaces/', parameters('privateEndpoints')[copyIndex()].privateEndpointConfiguration.privateEndpoint.name, '-nic')]"
+                                    },
+                                    "nicRecordsTemplateUri": {
+                                        "value": "https://go.microsoft.com/fwlink/?linkid=2264719"
+                                    },
+                                    "ipConfigRecordsTemplateUri": {
+                                        "value": "https://go.microsoft.com/fwlink/?linkid=2265018"
+                                    },
+                                    "uniqueId": {
+                                        "value": "[parameters('uniqueId')]"
+                                    },
+                                    "existingRecords": {
+                                        "value": {}
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "type": "Microsoft.Network/privateEndpoints/privateDnsZoneGroups",
+                            "apiVersion": "2020-03-01",
+                            "name": "[concat(parameters('privateEndpoints')[copyIndex()].privateEndpointConfiguration.privateEndpoint.name, '/', 'default')]",
+                            "location": "[parameters('location')]",
+                            "dependsOn": [
+                                "[parameters('privateDnsZone')]"
+                            ],
+                            "properties": {
+                                "privateDnsZoneConfigs": [
+                                    {
+                                        "name": "privatelink-cognitiveservices",
+                                        "properties": {
+                                            "privateDnsZoneId": "[concat(parameters('resourceGroupId'), '/providers/Microsoft.Network/privateDnsZones/', parameters('privateDnsZone'))]"
+                                        }
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            }
+        }
+    ],
+    "outputs": {
+        "AZURE_CLIENT_SECRET": {
+            "type": "string",
+            "value": "['foo']"
+        },
+        "MODEL_ENDPOINT": {
+          "type": "string",
+          "value": "['https://java-glharper-aoai.openai.azure.com/openai/deployments/gpt-4o/']"
+        }
+    }
+}
```

Comment:
Do we need the empty tests.yml file below? (can't comment on the empty file, so, commenting here)